modules/apps/27-interchain-accounts/keeper/keeper_test.go

```diff
@@ -4,21 +4,26 @@ import (
 	"fmt"
 	"testing"
 
+	sdk "github.com/cosmos/cosmos-sdk/types"
 	authtypes "github.com/cosmos/cosmos-sdk/x/auth/types"
 	"github.com/stretchr/testify/suite"
+	"github.com/tendermint/tendermint/crypto"
 
 	"github.com/cosmos/ibc-go/modules/apps/27-interchain-accounts/types"
 	channeltypes "github.com/cosmos/ibc-go/modules/core/04-channel/types"
 	ibctesting "github.com/cosmos/ibc-go/testing"
 )
 
 var (
+	// TestAccAddress defines a resuable bech32 address for testing purposes
+	// TODO: update crypto.AddressHash() when sdk uses address.Module()
+	TestAccAddress = types.GenerateAddress(sdk.AccAddress(crypto.AddressHash([]byte(types.ModuleName))), TestPortID)
```

Comment:
Presumably the sdk will eventually be using `address.Module()` from ADR-28 here: https://github.com/cosmos/cosmos-sdk/blob/master/x/auth/types/account.go#L164

So we'll need to replace `crypto.AddressHash()` when that's added

Reply:
Could we open an issue to track this? Should we open an issue on the SDK?

Reply:
Sure, I can open an issue on the SDK for this. I'll cc you